upgrades-docker-migrate.adoc

```diff
@@ -0,0 +1,57 @@
+= Migrate to new docker settings
+:toc: left
+:toclevels: 1
+:numbered:
+:source-highlighter: pygments
+:pygments-style: friendly
+
+While `taiga-docker` has been a pretty successful way for many people to have a Taiga self-hosted, the Taiga team receives a bunch of questions about the docker configuration. We have released a new `taiga-docker` version with the main settings based in an `.env` file. The main goal behind this change is to make it way easier to have a basi Taiga up and running.
+
+For those of you who already have a self-hosted Taiga running with docker, this tutorial will guide you through the steps of the migration.
+
+[[taigadckr-to-taigadckr]]
+== Summary
+
+If you are familiar with docker and docker compose, this is the sensible summary:
+- save the modifications made in the `docker-compose.yml` and `docker-compose-inits.yml` files
```

Comment:
I don't know if it deserves to warn about these files being overwritten, or if it'd be redundant.